### PR TITLE
Email summary content lead

### PR DESCRIPTION
**Pull request by jrgant**
I added a new exported function called `get_content_lead()` to retrieve the current content lead's email based on the current day. The function adds to the `send_summary_to` vector in `inst/email_summary.R`.

### CODE REVIEW

**Comment by jrgant** [comment]
@malcolmbarrett This contains all the changes reflected in my prior pull request. Apologies, as I thought this new branch I created would only contain the changes I made related to the updated email summaries.

Let me know if you need me to do anything differently, since the other pull request (for the tweet library processing) should be handled first.

**Comment by malcolmbarrett** [comment]
In future PRs, use `usethis::pr_init()`, which will prevent that problem

File: R/email_summary.R [hunk 1]
```diff
@@ -23,6 +23,37 @@ get_ny_tz <- function() {
...
+#' Retrieve Email for Active Content Lead
+#'
+#' `get_content_lead()` retrieves the email of teh currently assigned content
+#' lead. The function retrieves a list of content leads and their assignment
+#' dates, checks these against the current date, and returns a character string
+#' containing an email address.
+#' @param google_drive_auth Authorization token for the SER Google Drive account.
+#' @export
+get_content_lead <- function(google_drive_auth = drive_auth_token()) {
+  require(lubridate)
```
Comment (malcolmbarrett) [inline R/email_summary.R:35]: You should never use `library()` or `require()` in R package code.

Instead, if you need to call functions without pkg::, use `@importFrom`. Here, you need to add this to the roxygen: `@importFrom lubridate today %within% interval mdy`. 

However, my preference here is that you limit the import to `importFrom lubridate %within%` and use `lubridate::` for the rest.

**Review by malcolmbarrett — CHANGES_REQUESTED** [review]
Outside of the comment I left, this looks good. You have your work cut out for you managing this merge, though, not working from a clean branch. You may be better off rebasing after merging #10

**Comment by malcolmbarrett** [comment]
Closing this in favor of #17 rather than doing git fu to get out of this merge situation

**Pull request closed without merging**